lib/sidekiq/batch/web.rb

```diff
@@ -0,0 +1,44 @@
+require 'sidekiq/web' unless defined?(Sidekiq::Web)
+
+module Sidekiq
+  class Batch
+    module Web
+
+      LOCALE_PATH = Pathname.new(__dir__).join("web/locales")
+      VIEW_PATH = Pathname.new(__dir__).join("web/views")
```

Comment:
```suggestion
      LOCALE_PATH = Pathname.new(__dir__).join('web/locales')
      VIEW_PATH = Pathname.new(__dir__).join('web/views')
```

nit 😜 
(several places like that in the file)

Reply:
I tend to always use `double quotes` for everything https://anti-pattern.com/always-use-double-quoted-strings-in-ruby but I can adjust it if we have a policy to use a single quote for a string that does not require interpolation.

Reply:
We do, see here for the `standards` policy that we enforce on the other repos
https://github.com/distribusion/standards/blob/master/configurations/.rubocop.standards.yml#L127-L128

Reply:
> As this is not public, maybe you can format locally using our standards?

I will do it but on a separate branch because I wanted to keep this one as a source of my pull request to the original `sidekiq-batch`